atom/browser/node_debugger.h

```diff
@@ -26,7 +24,7 @@ class NodeDebugger {
 
  private:
   node::Environment* env_;
-  std::unique_ptr<node::NodePlatform> platform_;
+  node::NodePlatform* platform_;
```

Comment:
@zcbenz `platform_` will be uninitialized if inspector was not enabled, causing destructor to crash.

Reply:
@deepak1556 Thanks for finding it, will fix it.